lib/matplotlib/backends/backend_gtk3.py

```diff
@@ -814,20 +811,20 @@ def _call_tool(self, btn, name):
     def set_message(self, s):
         self.message.set_label(s)
 
-    def toggle_toolitem(self, name):
+    def toggle_toolitem(self, name, toggled):
         if name not in self._toolitems:
             return
-
-        status = self._toolitems[name].get_active()
-        self._toolitems[name].handler_block(self._signals[name])
-        self._toolitems[name].set_active(not status)
-        self._toolitems[name].handler_unblock(self._signals[name])
+        for toolitem, signal in self._toolitems[name]:
```

Comment:
ups, sorry, forget about this comment. I didn't realize we were allowing multiple times the same tool